burr/tracking/server/run.py

```diff
@@ -82,6 +82,18 @@ async def ready() -> bool:
     return True
 
 
+@app.get("/api/v0/version")
```

Comment:
This is only useful for the demos, right? Otherwise `burr` will not be included, as it is not needed. 

Add a comment on use-case.

Reply:
What do you mean not included? That's how the server is installed?

Reply:
In production contexts, `burr` will not be installed in the server. That said, we have no production deployments yet...